src/components/Modal/ModalFooter/ModalFooter.scss

```diff
@@ -0,0 +1,11 @@
+.moonstone-modal-footer {
+    display: flex;
+    gap: var(--spacing-medium);
+    justify-content: end;
```

Comment:
I prefer to use Moonstone utilities classes `FlexRow_reverse`, `alignCenter` to manage flex display to prevent writing the same CSS for each component.